pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/index.md

```diff
@@ -0,0 +1,108 @@
+---
+$title: Integrate your technology with AMP
+$order: 0
+$hidden: true
+description: 'If you are a technology provider for publishers or advertisers on the web, we invite you to add support to AMP so your customers can continue to leverage your technology and ...'
+formats:
+  - websites
+  - ads
+  - stories
+  - email
+---
+
+Thanks for your interest in contributing to AMP! We appreciate your participation in making the web a user forward platform. 
+
+Publishers have created over 1.4B AMP documents that are hosted on over 750K unique domains. Such growth would have been impossible without strong support from over 100 third party tech companies that have already integrated with AMP.
+
+If you are a technology provider for publishers or advertisers on the web, we invite you to add support to AMP so your customers can continue to leverage your technology and achieve our joint vision to build a better web.
+
+This document outlines third party vendor expectations and defines contribution levels. 
+
+# Contributing guidelines
+
+All general contributions are subject to the [AMPHTML universal guidelines in CONTRIBUTING.md](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md). We expect vendors to test, maintain, and update their contribution to various degrees.
+
+To qualify for inclusion, all contribution levels must:
+
+*   Meet the [notability requirements of the English Wikipedia](https://en.wikipedia.org/wiki/Wikipedia:Notability).
+*   Maintain or improve the same type of service level as AMP promises to both publishers and users. 
+*   Be created in good quality.
+*   Create a troubleshooting channel for their customers. 
+*   Provide good integration test coverage from their side to test against both AMP’s prod & canary releases.
+*   Fulfill a purpose that does not exist. 
+
+Vendor contributions are divided into three different levels. Each level is dependent on how much additional business logic and vendor specific logic (logic that is not shared with other vendors) is added to the AMP repo. A general principle is that the more logic, especially vendor specific logic, added to AMP, the higher contribution level is and the more commitment needed from the vendor.
+
+Level one & level two contributions share components between vendors. If there is a component fulfills a purpose similar to your business, consider reusing that component. This requires much less effort and is more long-term maintainable.
```

Comment:
Hm. Have you considered using number literals for the different levels? `Level 1`, `Level 2`. These make it easier to parse the text.

Reply:
That's the journalist in me - trained to write out numbers until 10. I'll update it